asdf/tests/test_fits_embed.py

```diff
@@ -275,33 +275,6 @@ def test_asdf_in_fits_open(tmpdir, dtype):
             compare_asdfs(asdf_in_fits, ff)
 
 
-@pytest.mark.parametrize('dtype', TEST_DTYPES)
-def test_asdf_open(tmpdir, dtype):
-    """Test the top-level open method of the asdf module"""
-    tmpfile = os.path.join(str(tmpdir), 'test.fits')
-    # Write the AsdfInFits object out as a FITS file with ASDF extension
-    asdf_in_fits = create_asdf_in_fits(dtype)
-    asdf_in_fits.write_to(tmpfile)
-
-    # Test opening the file directly from the URI
-    with asdf_open(tmpfile) as ff:
-        compare_asdfs(asdf_in_fits, ff)
-
-    # Test open/close without context handler
-    ff = asdf_open(tmpfile)
-    compare_asdfs(asdf_in_fits, ff)
-    ff.close()
```

Comment:
Shouldn't these still be tested, as they are still supported?

We're only getting rid of reading `fits.gz`, open `HDUList` objects, right?  Also what about passing an open file handle of the FITS file?  That is not mentioned above as something that is being removed.  Is it?